### ISSUE TITLE
Cannot display cpu/mem/swap and other information.

### ISSUE THREAD

**Issue by Rougnt** [arcan1s/awesome-widgets#158]
After build from 3.5.1. I cannot see any information.
When I try to see cpu value in setting, I only saw 
```
tag: cpu 
value: %1(I18N_ARGUMENT_MISSING
information: (none)
```
system is kubuntu22.04
plasma 5.24.7
QT 5.15.3

**Comment by arcan1s** [comment]
hmm, do you mean in notification window or do values also disappear in widget itself?

**Comment by arcan1s** [comment]
it appears that you don't have `ksysguard` installed, which provides common meters (and looks like package recipe has to be updated). Unfortunately it looks like the package has been removed from [ubuntu repositories](https://www.reddit.com/r/Kubuntu/comments/ukcj02/what_happened_to_ksysguard_for_kubuntu_2204/), probably it is time to migrate to new ksystemstats :(

**Comment by arcan1s** [comment]
closed as part of #162
